src/hooks/UseRequirementData/UseRequirementData.js

```diff
@@ -45,125 +46,181 @@ const UseRequirementData = (files) => {
     };
     
     // post given file
-    // const postFile = async (file, index) => {
-    //     try {
-    //         const base64Content = await fileToBase64(file);
-    //         const response = await axios.post(createURL(), {
+    const postFile = async (file, index) => {
+        try {
+            const base64Content = await fileToBase64(file);
+            const response = await axios.post(createURL(), {
+                mode: 'no-cors',
+                method: 'post',
+                headers: {
+                    Accept: 'application/json'
+                },
+                body: base64Content
```

Comment:
the body now consist of two keys one encoded_pdf and file_name and has to be converted into json string:
```javascript 
body: JSON.stringify({
                 'encoded_pdf': fileToBase64(file),
                 'file_name': file.name
             })  
```